subworkflows/local/depths.nf

```diff
@@ -0,0 +1,65 @@
+params.mag_depths_options                           = [:]
+params.mag_depths_plot_options                      = [:]
+params.mag_depths_summary_options                   = [:]
```

Comment:
what are those for, seem not be used?

Reply:
I will have to check, that was carried over from the original code I imagine.

Reply:
Don't find any reference to them anywhere else :+1: